docs/release_notes/v1.0.0.md

```diff
@@ -0,0 +1,84 @@
+
+  - [Delta Lake](https://spiceai.org/docs/components/data-connectors/delta-lake)
+  - [MySQL](https://spiceai.org/docs/components/data-connectors/mysql)
+  - [Dremio](https://spiceai.org/docs/components/data-connectors/dremio)
+  - [PostgreSQL](https://spiceai.org/docs/components/data-connectors/postgres)
+  - [Databricks (mode: delta_lake)](https://spiceai.org/docs/components/data-connectors/databricks)
+  - [DuckDB](https://spiceai.org/docs/components/data-connectors/duckdb)
+  - [S3](https://spiceai.org/docs/components/data-connectors/s3)
+
+- **Stable Data Accelerators:** The following data accelerators graduated to [Stable](https://github.com/spiceai/spiceai/blob/trunk/docs/criteria/accelerators/stable.md):
+
+  - [DuckDB](https://spiceai.org/docs/components/data-accelerators/duckdb)
+  - [Arrow](https://spiceai.org/docs/components/data-accelerators/arrow)
+
+- **Unity Catalog Connector:** Graduated to [Stable](https://github.com/spiceai/spiceai/blob/trunk/docs/criteria/catalogs/stable.md).
+
+- **Databricks (mode: spark_connect) Data Connector:** Graduated to [Beta](https://github.com/spiceai/spiceai/blob/trunk/docs/criteria/connectors/beta.md).
+
+- **Beta Catalog Connectors:** The Iceberg and Databricks catalog connectors graduated to [Beta](https://github.com/spiceai/spiceai/blob/trunk/docs/criteria/catalogs/beta.md).
+
+- **OpenAI Model & Embeddings Provider:** Graduated to [Release Candidate (RC)](https://github.com/spiceai/spiceai/blob/trunk/docs/criteria/models/rc.md).
+
+- **Alpha Model Providers:** The Anthropic and xAI (Grok) model providers graduated to [Alpha](https://github.com/spiceai/spiceai/blob/trunk/docs/criteria/models/alpha.md).
+
+## Breaking Changes
+
+- **Identifier Normalization:** Unquoted identifiers such as table names are no longer normalized to lowercase. Identifiers will now retain their exact case as provided.
+
+- **Sandboxed Docker Image:** The Runtime Docker Image now runs the `spiced` process as the `nobody` user in a minimal [chroot sandbox](https://www.oreilly.com/library/view/network-security-hacks/0596006438/ch01s11.html).
+
+- **Insecure S3 and ABFS endpoints:** The S3 and ABFS connectors now enforce insecure endpoint checks, preventing HTTP endpoints unless `allow_http` is explicitly enabled. Refer to [the documentation](https://spiceai.org/docs/components/data-connectors/s3#params) for details.
+
+## Dependencies
+
+No major dependency changes.
+
+## Upgrading
+
+To upgrade to v1.0.0, use one of the following methods:
+
+**CLI**:
+
+```console
+spice upgrade
+```
+
+**Homebrew**:
+
+```console
+brew upgrade spiceai/spiceai/spice
+```
+
+**Docker**:
+
+Pull the spiceai/spiceai:1.0.0 image:
+
+```console
+docker pull spiceai/spiceai:1.0.0
+```
+
+For available tags, see [DockerHub](https://hub.docker.com/r/spiceai/spiceai/tags).
+
+**Helm**:
+
+```console
+helm upgrade spice ./charts/spice --version 1.0.0
```

Comment:
```suggestion
helm repo update
helm upgrade spiceai spiceai
```

Reply:
This keeps the same syntax that we document: https://spiceai.org/docs/deployment/kubernetes